draft-ietf-httpbis-messaging-latest.xml

```diff
@@ -2180,7 +2186,8 @@ Connection: close
 <t>
    Extensions to the protocol might be used to mitigate the risk of unwanted
    modification of messages by intermediaries, even when the "https" scheme is
-   used. Integrity might be assured by using hash functions or digital
+   used. Integrity might be assured by using message authentication codes
+   or digital
```

Comment:
Hash functions do not provide integrity protection in the face of modification of messages by intermediaries.